src/libs/TransactionUtils/index.ts

```diff
@@ -1219,88 +1230,89 @@ function buildTransactionsMergeParams(reviewDuplicates: OnyxEntry<ReviewDuplicat
 }
 
 export {
+    allHavePendingRTERViolation,
```

Comment:
All these changes are unnecessary and it might just cause conflicts when cherry picked to staging, can you please revert them?